docs/design-proposals/2020-06-12-heterogeneous-design-for-tidb-cluister.md

```diff
@@ -1,4 +1,4 @@
-# Heterogeneous design for TidbCluster
+# Deprecated - Heterogeneous design for TidbCluster
```

Comment:
```suggestion
# Heterogeneous design for TidbCluster (DEPRECATED)
```

Reply:
can you rename this file to `2020-06-12-heterogeneous-design-for-tidb-cluster.md`, it has a typo in the file name

Reply:
done